tests/layer/test_rgcn.py

```diff
@@ -0,0 +1,322 @@
+import numpy as np
+import networkx as nx
+from stellargraph.layer.rgcn import RelationalGraphConvolution, RGCN
+from stellargraph.mapper.node_mappers import RelationalFullBatchNodeGenerator
+import pytest
+from scipy import sparse as sps
+from stellargraph.core.utils import normalize_adj
+from tensorflow import keras
+from tensorflow.keras import backend as K
+from tensorflow.keras.layers import Input, Lambda
+from stellargraph import StellarDiGraph
+from stellargraph.layer.misc import SqueezedSparseConversion
+import pandas as pd
+
+
+def create_graph_features():
+    G = nx.MultiDiGraph()
+    G.add_nodes_from(["a", "b", "c"])
+    G.add_edges_from([("a", "b", "r1"), ("b", "c", "r1"), ("a", "c", "r2")])
+    return G, np.array([[1, 1], [1, 0], [0, 1]])
+
+
+def test_RelationalGraphConvolution_config():
+    rgcn_layer = RelationalGraphConvolution(units=16, num_relationships=5)
+    conf = rgcn_layer.get_config()
+
+    assert conf["units"] == 16
+    assert conf["activation"] == "linear"
+    assert conf["num_bases"] == 0
+    assert conf["num_relationships"] == 5
+    assert conf["use_bias"] == True
+    assert conf["kernel_initializer"]["class_name"] == "GlorotUniform"
+    assert conf["bias_initializer"]["class_name"] == "Zeros"
+    assert conf["kernel_regularizer"] == None
```

Comment:
I think comparing for equality with None should use `is` not `==`.

Reply:
You're absolutely right, fixed!